website/content/docs/discovery/dns.mdx

```diff
@@ -136,17 +136,22 @@ it is recommended to use the HTTP API to retrieve the list of nodes.
 The format of a standard service lookup is:
 
 ```text
-[<tag>.]<service>.service[.<datacenter>].<domain>
+[<tag>.]<service>.service[.<datacenter-or-peer>].<domain>
```

Comment:
This is not supported, because it's never passed in as a peer here: https://github.com/hashicorp/consul/blob/main/agent/dns.go#L765
Its only passed in as a datacenter.

Reply:
This would be something we intend to support in the future.

Reply:
If I understand correctly, this format is not supported, but it is now possible to pass a peer as of https://github.com/hashicorp/consul/pull/15596

If I understand correctly, the formats are now:
- `[<tag>.]<service>.service[.<datacenter>].<domain>`
- `[<tag>.]<service>.service[.<datacenter>].dc.<domain>`
- `[<tag>.]<service>.service[.<peer>].peer.<domain>`

Reply:
@hashi-derek : Is it accurate to say that `[<tag>.]<service>.service[.<datacenter-or-peer>].<domain>` isn't supported, because in that form, the `<datacenter-or-peer>` label is always parsed as a datacenter?

Reply:
Gotcha. So `datacenter-or-peer` should not appear **anywhere** in these docs.

Reply:
Correct, unless there's an edge case that I'm not accounting for. To my knowledge, I avoided any potential for that to occur.